packages/cmf/package.json

```diff
@@ -32,6 +32,32 @@
   "bin": {
     "cmf-settings": "./scripts/cmf-settings.js"
   },
+  "dependencies": {
+    "@babel/polyfill": "^7.0.0",
+    "ajv": "^6.2.1",
```

Comment:
Could we remove this one, & also the unused ajv in https://github.com/Talend/ui/blob/master/packages/cmf/scripts/cmf-settings.utils.js#L75